documents4j-transformer-msoffice/documents4j-transformer-msoffice-base/src/main/java/com/documents4j/conversion/msoffice/AbstractMicrosoftOfficeBridge.java

```diff
@@ -64,15 +64,20 @@ protected StartedProcess doStartConversion(File source, DocumentType sourceType,
         try {
             MicrosoftOfficeFormat microsoftOfficeFormat = formatOf(targetType);
             // Always call destroyOnExit before adding a listener: https://github.com/zeroturnaround/zt-exec/issues/14
+            String[] command = {"cmd", "/S", "/C",
+                    doubleQuote(conversionScript.getAbsolutePath(),
+                            source.getAbsolutePath(),
+                            target.getAbsolutePath(),
+                            microsoftOfficeFormat.getValue())};
+
+            getLogger().trace("Running command for conversion " + command.toString());
```

Comment:
This won't log anything meaningful. Please use the log placeholder notation *Running command for conversion {}* and wrap the array in `Arrays.toString()` to have the log print something.

Reply:
Done